src/go/rdctl/pkg/snapshot/snapshotter_windows.go

```diff
@@ -0,0 +1,120 @@
+package snapshot
+
+import (
+	"fmt"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/factoryreset"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/wsl"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+type wslDistro struct {
+	// The name of the WSL distro.
+	Name string
+	// The path to the directory that is used to store the
+	// copy of the distro that is actually used by WSL.
+	WorkingDirPath string
+}
+
+func getWslDistros(paths paths.Paths) []wslDistro {
```

Comment:
I'm tempted to switch it to a `map[string]string`, but yes, it is nice to have the documentation on the properties. Mostly because `WorkingDirPath` is a little confusing: it is not the path to the filesystem image, but rather the path to the parent directory of the filesystem image.